### PR TITLE
Atmar21

### PR DESCRIPTION
**Pull request by zachandfox**
[10.1101/2021.10.10.21264827 #387](https://github.com/hivdb/covid-drdb-payload/issues/387)

### CODE REVIEW

File: excluded/rx_plasma/atmar21-plasma.csv [hunk 1]
```diff
@@ -0,0 +1,28 @@
+ref_name,subject_name,rx_name,collection_date_cmp,collection_date,location,cumulative_group,section
```
Comment (KaimingTao) [inline excluded/rx_plasma/atmar21-plasma.csv:1]: @zachandfox what's the reason to exclude these data?

File: tables/rx_potency/atmar21-pot.csv [hunk 2]
```diff
@@ -0,0 +1,151 @@
+﻿ref_name,rx_name,iso_name,section,assay_name,potency_type,potency,cumulative_count,potency_upper_limit,potency_lower_limit,potency_unit,date_added
+Atmar21,Ad26COV2S_2dose_mRNA1273_1dose_1day,WA1 Spike,Table 2,Pseudovirus (lentivirus),NT50,57,51,45118,10,NULL,2022-06-21
```
Comment (KaimingTao) [inline tables/rx_potency/atmar21-pot.csv:2]: The EUA standard of Ad26.COV.2 is 1 dose.

**Review by KaimingTao — CHANGES_REQUESTED** [review]

File: excluded/rx_plasma/atmar21-plasma.csv (hunk 1 above)
Comment (zachandfox) [reply excluded/rx_plasma/atmar21-plasma.csv:1]: These do not have a second isolate to compare against B.1, as pointed out by the CI check on [a13c63b](https://github.com/hivdb/covid-drdb-payload/runs/6991450877?check_suite_focus=true)

File: tables/rx_potency/atmar21-pot.csv (hunk 2 above)
Comment (zachandfox) [reply tables/rx_potency/atmar21-pot.csv:2]: Ah, I'll change this then

File: excluded/rx_plasma/atmar21-plasma.csv (hunk 1 above)
Comment (KaimingTao) [reply excluded/rx_plasma/atmar21-plasma.csv:1]: I see, thanks for pointing out.

File: tables/rx_potency/atmar21-pot.csv (hunk 2 above)
Comment (zachandfox) [reply tables/rx_potency/atmar21-pot.csv]: Are the changes around this correct?

**Pull request merged**